cmake/emu/qemu.cmake

```diff
@@ -226,11 +226,14 @@ elseif(QEMU_NET_STACK)
     set_ifndef(NET_TOOLS ${ZEPHYR_BASE}/../net-tools) # Default if not set
 
     list(APPEND PRE_QEMU_COMMANDS_FOR_server
-      COMMAND ${NET_TOOLS}/monitor_15_4
-  ${PCAP}
-  /tmp/ip-stack-server
+      COMMAND
+      {
```

Comment:
could you add a comment on why this is needed.

Reply:
> could you add a comment on why this is needed.

Sorry, this should have been a `request-changes`, otherwise the fix is approved.